system/lib/libc/emscripten_memset.c

```diff
@@ -9,16 +9,7 @@ __attribute__((no_sanitize("address"))) void *__memset(void *str, int c, size_t
 __attribute__((__weak__)) void *__musl_memset(void *str, int c, size_t n);
 __attribute__((__weak__)) void *__memset(void *str, int c, size_t n);
 
-#ifdef EMSCRIPTEN_OPTIMIZE_FOR_OZ
-
-void *__memset(void *str, int c, size_t n) {
-  unsigned char *s = (unsigned char *)str;
-#pragma clang loop unroll(disable)
-  while(n--) *s++ = c;
-  return str;
-}
-
-#elif defined(__wasm_bulk_memory__)
+#if defined(__wasm_bulk_memory__) || defined(EMSCRIPTEN_OPTIMIZE_FOR_OZ)
```

Comment:
Ditto, remove first clause?   (maybe #error if __wasm_bulk_memory__ is bulk memory is not defined to be sure).